### PR TITLE
[gh-pages]: refactor svg assets and collection pages

### PR DESCRIPTION
**Pull request by Natalie-Smirnova**
- Examples index page:  SVG representation picture on the page was updated, the draft pages partially transparent on the index page was made ,  badges were added.
- Core and Components index page:  the grid style was replaced with the list style, badges were added.
- The list item UI was updated : badges were added, white background,  icons on hover, two columns view 
image
image
list
examples

### CODE REVIEW

File: pages/views/core/esl-utils.njk [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: ESL Utils
 seoTitle: Universal JS, TS and Web API utilities set from Exadel Smart Library
 name: ESL Utils
 tags: core
-iconAlt: '@bind @memoize @prop'
+icon: 'common/right-arrow.svg'
```
Comment (ala-n) [inline pages/views/core/esl-utils.njk:7]: It looks like we don't need to set this information to pages directly, it is common styling

File: pages/static/assets/sidebar/components.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...375 1.917a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h19.25a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM21.625 14.75H9.71a1.377 1.377 0 0 1-1.375-1.375v-2.75c0-.758.616-1.375 1.375-1.375h11.916c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM9.71 10.167a.458.458 0 0 0-.459.458v2.75c0 .253.206.458.459.458h11.916a.458.458 0 0 0 .459-.458v-2.75a.458.458 0 0 0-.459-.458H9.71ZM3.75 14.75A2.753 2.753 0 0 1 1 12a2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 6.5 12a2.753 2.753 0 0 1-2.75 2.75Zm0-4.583A1.835 1.835 0 0 0 1.917 12c0 1.011.822 1.833 1.833 1.833A1.835 1.835 0 0 0 5.583 12a1.835 1.835 0 0 0-1.833-1.833ZM14.292 23H2.375A1.377 1.377 0 0 1 1 21.625v-2.75c0-.758.617-1.375 1.375-1.375h11.917c.758 0 1.375.617 1.375 1.375v2.75c0 .758-.617 1.375-1.375 1.375ZM2.375 18.417a.458.458 0 0 0-.458.458v2.75c0 .253.205.458.458.458h11.917a.458.458 0 0 0 .458-.458v-2.75a.458.458 0 0 0-.458-.458H2.375ZM20.25 23a2.753 2.753 0 0 1-2.75-2.75 2.753 2.753 0 0 1 2.75-2.75A2.753 2.753 0 0 1 23 20.25 2.753 2.753 0 0 1 20.25 23Zm0-4.583a1.835 1.835 0 0 0-1.833 1.833c0 1.011.822 1.833 1.833 1.833a1.835 1.835 0 0 0 1.833-1.833 1.835 1.835 0 0 0-1.833-1.833Z"/></svg>
```
Comment (ala-n) [inline pages/static/assets/sidebar/components.svg:1]: It looks like it's better to remove `fill="none"` as well as we have no icon on preview in editors, which is not good

File: pages/src/collection-grid/collection-grid.less [hunk 3]
```diff
@@ -56,26 +73,45 @@
...
+      display: none;
+    }
 
+    &:hover  {
+      svg {
+        display: block;
+        fill: @primary-blue;
+      }
+      border-color: @primary-blue;
+    }
```
Comment (ala-n) [inline pages/src/collection-grid/collection-grid.less:104]: I'd recommend to replace arrow SVG with a CSS alternative with a pseudo-element and border embedded svg

File: pages/views/components/esl-select-list.njk [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: ESL Select List
 seoTitle: ESL Select List - web component for custom select list UI
 name: ESL Select List
 tags: [components, forms, beta]
-iconAlt: '<esl-select-list/>'
+icon: 'common/right-arrow.svg'
```
[the remark this answers is not in the document]
Comment (ala-n) [reply pages/views/components/esl-select-list.njk:7]: And also, an important note we definitely don't want to have an icon of example to be presented in the title on the page

**Review by ala-n — CHANGES_REQUESTED** [review]

File: pages/views/_layouts/content.njk [hunk 5]
```diff
@@ -9,6 +9,7 @@
     <div class="content {{ containerCls or 'container' }} {{ 'container-aside' if aside }}">
       {% if title %}
         <h1 class="page-title">
+          {% if icon and not aside %}{% include "static/assets/" + icon %}{% endif %}
```
Comment (ala-n) [inline pages/views/_layouts/content.njk:12]: `aside` is not a good criterion to define if we need an icon, 
As a suggestion, try to rename icon assets for samples pages to `preview` then we will not have a conflict with that meta fields

**Review by ala-n — COMMENTED** [review]
Looks cool!
Just a couple of last cosmetic suggestions:
1) Make a draft badge red on collections pages
2) As for me, it's better to render the draft badge in a list view in addition to opacity

File: pages/views/_layouts/content.njk [hunk 6]
```diff
@@ -9,6 +9,7 @@
     <div class="content {{ containerCls or 'container' }} {{ 'container-aside' if aside }}">
       {% if title %}
         <h1 class="page-title">
+          {% if preview %}{% include "static/assets/" + preview %}{% endif %}
```
Comment (ala-n) [inline pages/views/_layouts/content.njk:12]: Sorry but the `preview` for the icon looks strange, I meant that we could rename the field for examples collection.
If we are going to change that field, we can consider `sectionIcon`,  `collectionIcon`, or smth like that

**Review by ala-n — APPROVED** [review]

File: pages/src/collection-grid/collection-grid.less [hunk 7]
```diff
@@ -14,28 +14,46 @@
...
     display: flex;
     align-items: center;
     justify-content: center;
     width: 100%;
+    transition: opacity .8s;
+  }
+
+  &-item.draft:not(:hover) &-box {
+    opacity: 30%;
+
```
Comment (fshovchko) [inline pages/src/collection-grid/collection-grid.less:38]:
```suggestion
```

File: pages/src/collection-grid/collection-grid.less [hunk 8]
```diff
@@ -14,28 +14,46 @@
...
+  }
```
Comment (fshovchko) [inline pages/src/collection-grid/collection-grid.less:39]:
```suggestion
    opacity: 30%;
  }
```

**Comment by ala-n** [comment]
TODO will be moved to a separate request.
Merging this update with squash

**Pull request merged**